soc/arm/nordic_nrf/nrf91/dts_fixup.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* SoC level DTS fixup file */
+
+#define DT_NUM_IRQ_PRIO_BITS        DT_ARM_V8M_NVIC_E000E100_ARM_NUM_IRQ_PRIORITY_BITS
+
+#define DT_ADC_0_NAME                   DT_NORDIC_NRF_SAADC_ADC_0_LABE
+
+#define DT_UART_0_NAME			DT_NORDIC_NRF_UARTE_UART_0_LABEL
+#define DT_UART_1_NAME			DT_NORDIC_NRF_UARTE_UART_1_LABEL
+#define DT_UART_2_NAME			DT_NORDIC_NRF_UARTE_UART_2_LABEL
+#define DT_UART_3_NAME			DT_NORDIC_NRF_UARTE_UART_3_LABEL
+
+#define DT_FLASH_DEV_NAME \
+	DT_NRF_NRF91_FLASH_CONTROLLER_FLASH_CONTROLLER_LABEL
```

Comment:
I am unable to find `DT_NRF_NRF91_FLASH_CONTROLLER_FLASH_CONTROLLER_LABEL` in a `generated_dts_board.h` file for a project built for `nRF9160_pca10090`. Could you double check that this is correct?

Reply:
This is a nice comment, thanks. So, this has been incorrect in downstream. where the fixup was defining `DT_FLASH_DEV_NAME `as `DT_NRF_NRF52_FLASH_CONTROLLER_4001E000_LABEL`. This is incorrect, not only for the "52" in-fix in the name, but also because of wrong address label - it may be 4001E000 or 5001E000, depending on security domain.

This has been already fixed in upstream, after merging the nrf91 DTS support. Therefore, we will now be able to generate the correct define:  	`DT_NRF_NRF91_FLASH_CONTROLLER_FLASH_CONTROLLER_LABEL `